test/test_localTypes.py

```diff
@@ -45,7 +46,8 @@ def return_angle(pt1, pt2):
 
 class TestLocalTypes(unittest.TestCase):
 
```

Comment:
I would personally remove any empty lines between the class/function name and its docstring.